src/pages/ReplaceServer/DisableComputeServiceNetwork.js

```diff
@@ -339,18 +339,23 @@ class DisableComputeServiceNetwork extends BaseUpdateWizardPage {
       playbooks: [REMOVE_FROM_AGGREGATES]
     }];
 
-    if (this.props.operationProps.oldServer.isReachable && this.props.operationProps.server) {
+    if (this.props.operationProps.oldServer.isReachable &&
```

Comment:
Since two of the three conditions are the same between the if and else-if, it would be more straightforward to do something like:
```
if (this.props.operationProps.oldServer.hasInstances && this.props.operationProps.server) {
  if (this.props.operationProps.oldServer.isReachable) {
     steps.push(...some stuff)
  } else {
     steps.push(...that other stuff)
  }
}
```

Reply:
@GarySmith ok will update